packages/fxa-settings/src/components/InlineRecoveryKeySetupCreate/index.test.tsx

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { screen } from '@testing-library/react';
+import { renderWithLocalizationProvider } from 'fxa-react/lib/test-utils/localizationProvider';
+import { Subject } from './mocks';
+
+describe('InlineRecoveryKeySetupCreate', () => {
+  it('renders as expected', () => {
+    renderWithLocalizationProvider(<Subject />);
+
+    screen.getByLabelText('account recovery key', {
+      selector: 'svg',
+      exact: false,
+    });
+    screen.getByRole('heading', {
```

Comment:
The figma doc isn't clear what this page should be, it looks this this currently

![Screenshot 2024-09-06 at 1 46 33 PM](https://github.com/user-attachments/assets/e75082f1-fe73-4937-89c7-164f09892eeb)